examples/visual-tests/amp-story/amp-story-shopping.html

```diff
@@ -30,48 +30,51 @@
   <body>
     <amp-story
         standalone
-        title="Story Shopping Component Visual Diff Test"
+        title="Story Shopping Component"
         publisher="AMP Story Shopping"
         publisher-logo-src="example.com/logo.png"
         poster-portrait-src="example.com/poster.jpg">
       <amp-story-page id="inline-light-theme">
         <!--
-          Example of:
+          Example of
```

Comment:
Nit: you can put "Example of inline config" on a single line

Reply:
Done, also have added images that show how the single line, 2-line, and 2-line overflow cases appear on my local server.

Reply:
Update: Will add in images as soon as #37188 is in - that way it will more closely match the figma files.